### DISCUSSION TITLE
Arbitrarily modify battery charge level

### DISCUSSION THREAD

**Discussion by mousspi** [hif2k1/battery_sim#100] [Q&A]
Hi,
Thanks a lot for this perfect integration, just what I needed !

Is there a way to change battery charge level ?

I want to use it to check the virtual battery calculated by my energy provider.
My virtual battery is already at about 70kWh and I would like your simulated battery to start at the same point.
Is it possible ? how ?

Thansk a lot,
MoussPi.

**Comment by hif2k1** [comment]
I think you can edit the parameter of any sensor by going to developer tools > states and updating the value.
